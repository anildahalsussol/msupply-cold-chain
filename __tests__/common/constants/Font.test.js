import { FONT } from '~constants';

describe('Constants: FONT', () => {
  it('Matches snapshot', () => {
    expect(FONT).toMatchSnapshot();
  });
});
