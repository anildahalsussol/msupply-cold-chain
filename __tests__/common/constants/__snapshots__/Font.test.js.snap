// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: FONT Matches snapshot 1`] = `
Object {
  "ALIGN": Object {
    "CENTER": "center",
    "JUSTIFY": "justify",
    "LEFT": "left",
    "RIGHT": "right",
  },
  "FAMILY": Object {
    "BOLD": "Roboto-Bold",
    "MEDIUM": "Roboto-Medium",
    "REGULAR": "Roboto-Regular",
  },
  "SIZE": Object {
    "L": 40,
    "M": 26,
    "MS": 18,
    "S": 12,
    "XL": 70,
  },
  "TRANSFORM": Object {
    "CAPITALIZE": "capitalize",
    "LOWERCASE": "lowercase",
    "UPPERCASE": "uppercase",
  },
}
`;
